examples/svm/plot_custom_kernel.py

```diff
@@ -46,7 +46,7 @@ def my_kernel(X, Y):
 
 # Put the result into a color plot
 Z = Z.reshape(xx.shape)
-plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired)
+plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired, shading="nearest")
```

Comment:
This is great because it tackles a warning we have in the [notebook](https://scikit-learn.org/dev/auto_examples/svm/plot_custom_kernel.html):
```
/home/circleci/project/examples/svm/plot_custom_kernel.py:49: MatplotlibDeprecationWarning: shading='flat' when X and Y have the same dimensions as C is deprecated since 3.3.  Either specify the corners of the quadrilaterals with X and Y, or pass shading='auto', 'nearest' or 'gouraud', or set rcParams['pcolor.shading'].  This will become an error two minor releases later.
  plt.pcolormesh(xx, yy, Z, cmap=plt.cm.Paired)
```

Nonetheless, maybe this could go in another PR since it is not related to fixing the documentation to reflect the comments.